digitaltwins/device/src/digital_twin_client.ts

```diff
@@ -155,24 +139,31 @@ export class DigitalTwinClient {
   // The IoT Hub Twin that supports the Digital Twin concept.
   //
   private _twin: Twin;
-  constructor(capabilityModel: string, client: Client) {
-    /* Codes_SRS_NODE_DIGITAL_TWIN_DEVICE_06_001: [Will throw `ReferenceError` if `capabilityModel` argument is falsy.] */
-    if (!capabilityModel) throw new ReferenceError('capabilityModel must not be falsy');
+  constructor(client: Client) {
     /* Codes_SRS_NODE_DIGITAL_TWIN_DEVICE_06_002: [Will throw `ReferenceError` if the constructor `client` argument is falsy.] */
     if (!client) throw new ReferenceError('client must not be falsy');
-    this._capabilityModel = capabilityModel;
     this._client = client;
     this._twin = {} as Twin;
-    this.addInterfaceInstance(this._sdkInformation);
+    this._addInterfaceInstance(this._sdkInformation);
   }
 
   /**
-   * @method                        module:azure-iot-digitaltwins-device.DigitalTwinClient.addInterfaceInstance
-   * @description                   Adds the interfaceInstance to the Digital Twin client.  This will not cause
+   * @method                        module:azure-iot-digitaltwins-device.DigitalTwinClient.addInterfaceInstances
+   * @description                   Adds multiple interfaceInstances to the Digital Twin client.  This will not cause
    *                                any network activity.  This is a synchronous method.
-   * @param newInterfaceInstance            The object for a particular interfaceInstance.
+   * @param newInterfaceInstances   A single object or multiple objects for a particular interfaceInstance.
+   */
+  addInterfaceInstances(...args: BaseInterface[]) {
+    for (var i = 0; i < args.length; i++) {
+      this._addInterfaceInstance(args[i]);
+    }
+  }
```

Comment:
lol

Reply:
I mean... i kinda like how it reads

---
In reply to: [401161653](https://github.com/Azure/azure-iot-sdk-node/pull/738#discussion_r401161653) [](ancestors = 401161653)

Reply:
Oh I agree, it was just perfect.

---
In reply to: [401188298](https://github.com/Azure/azure-iot-sdk-node/pull/738#discussion_r401188298) [](ancestors = 401188298,401161653)